OWML.ModHelper.Menus/ModsMenu.cs

```diff
@@ -73,18 +74,28 @@ private void InitCombinationMenu(IModTabbedMenu options)
 
         private IModPopupMenu CreateModsMenu(IModTabbedMenu options)
         {
-            var modsTab = options.InputTab.Copy("MODS");
+            var modsTab = options.GameplayTab.Copy("MODS");
             modsTab.BaseButtons.ForEach(x => x.Hide());
             modsTab.Menu.GetComponentsInChildren<Selectable>(true).ToList().ForEach(x => x.gameObject.SetActive(false));
             modsTab.Menu.GetValue<TooltipDisplay>("_tooltipDisplay").GetComponent<Text>().color = Color.clear;
             options.AddTab(modsTab);
+            var firstDisabled = true;
+            var separator = new ModSeparator(modsTab) { Title = "ENABLED MODS" };
+            modsTab.AddSeparator(separator, 0);
+            separator.Element.transform.localScale = options.RebindingButton.Button.transform.localScale;
+            int index = 1;
             foreach (var modConfigMenu in _modConfigMenus)
             {
                 var modButton = options.RebindingButton.Copy(modConfigMenu.Manifest.Name);
+                if (!modConfigMenu.ModData.Config.Enabled && firstDisabled)
+                {
+                    modsTab.AddSeparator(separator.Copy("DISABLED MODS"), index++);
+                    firstDisabled = false;
+                }
                 modButton.Button.enabled = true;
                 InitConfigMenu(modConfigMenu, options);
                 modButton.OnClick += modConfigMenu.Open;
-                modsTab.AddButton(modButton);
+                modsTab.AddButton(modButton, index++);
```

Comment:
instead of all this code, could you 
1. get list of enabled mods
2. if any: add a label and the mods
3. get list if disabled mods
4. if any: add label and thrn those mods

use same incrementing index for all of these

Reply:
ah, yeah, I can just move creation of buttons block into a separate method

Reply:
should I remove sorting be `enabled` status from Owo now?

Reply:
i guess, it's no longer needed?